app/vlstorage/netselect/netselect.go

```diff
@@ -230,6 +231,14 @@ func (sn *storageNode) getStreamIDs(qctx *logstorage.QueryContext, limit uint64)
 	return sn.getValuesWithHits(qctx, "/internal/select/stream_ids", args)
 }
 
+func (sn *storageNode) getTenantIDs(ctx context.Context, start, end int64) ([]byte, error) {
+	args := url.Values{}
+	args.Set("start", fmt.Sprintf("%d", start))
+	args.Set("end", fmt.Sprintf("%d", end))
+	args.Set("disable_compression", fmt.Sprintf("%v", sn.s.disableCompression))
+	return sn.getResponseForPathAndArgs(ctx, "/internal/select/tenant_ids", args)
```

Comment:
It is better from readability and consistency PoV to parse the tenantIDs here and to return a `[]TenantID` instead of `[]byte`form this function.